api/components/com_languages/src/Controller/OverridesController.php

```diff
@@ -76,6 +76,24 @@ public function displayList()
         return parent::displayList();
     }
 
+    /** Method to edit a record - overwrites current value
+     * @param   integer  $recordKey  The primary key of the item (if exists)
```

Comment:
```suggestion
    /** Method to edit a record - overwrites current value
     *
     * @param   integer  $recordKey  The primary key of the item (if exists)
```

insert new line


also if you are editing an item then surely a primary key has to exist

Reply:
usually yes - but the language overrides are not in the database, they are in a text file... and the key is the identifier - not an integer, yes the comment should be adjusted,